_layouts/home.html

```diff
@@ -73,5 +298,128 @@ <h5 class="font-weight-bolder mt-3">Fully Responsive</h5>
     </div>
   </section>
 
-
-</div>
+  <section class="pt-6 pb-6" id="count-stats">
```

Comment:
I think the comments from here onward can be removed, unless you plan to use them as a reference.